src/client/src/index.js

```diff
@@ -19,12 +19,12 @@ const CaptionForm = require('./views/CaptionForm')
 m.route(document.body, '/start', {
   '/start': Start,
   '/scenes': Scenes,
-  '/edit-scene/:id': {
+  '/scenes/scene-:id': {
```

Comment:
Could you change :id to :sceneId like it is in the caption section

Reply:
Ok I fixed this.